docker/archive/dest.go

```diff
@@ -19,17 +19,32 @@ func newImageDestination(ctx *types.SystemContext, ref archiveReference) (types.
 	if ref.destinationRef == nil {
 		return nil, errors.Errorf("docker-archive: destination reference not supplied (must be of form <path>:<reference:tag>)")
 	}
-	fh, err := os.OpenFile(ref.path, os.O_WRONLY|os.O_EXCL|os.O_CREATE, 0644)
+	// tries to open a file first to accomodate for load when using stdout
+	// as the file is created first then written to
+	// if the file doesn't exist then it creates a new file for exclusive use
+	// if the file exists, it checks to ensure the size is 0
```

Comment:
One more thing - if there is an image(for example alpine) with multiple tags in docker and you do <docker save -o all.tar alpine> it saves all the tags into the archive. I guess that is why save has the cherry-picking option where you can choose which tag to save to the archive. 
This is not supported in containers/image. When a tag is not specified it defaults to the "latest" tag even if there are multiple tags of the same image.